### PR TITLE
fontWeight and fontStyle from textSymbolizer

### PR DESCRIPTION
**Pull request by pragmaeuge**
## Description

TextSymbolizer from Geostyler now includes also fontWeight and fontStyle properties, i have changed the getTextFont static method of OlStyleUtils.ts in order to includes weight and style in the OpenLayer text label.

## Related issues or pull requests

Until now, a Normal weight placeholder setting replaced the requested Geostyler text weight, and no fontStyle was provided.

## Pull request type

Please check the type of change your PR introduces:

- [x] Bugfix
- [ ] Feature
- [ ] Dependency updates
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe)
- [ ] I am unsure (we'll look into it together)

## Do you introduce a breaking change?

- [ ] Yes
- [ ] No
- [x] I am unsure (no worries, we'll find out)

## Checklist

- [x] I understand and agree that the changes in this PR will be licensed under the [BSD 2-Clause License](https://github.com/geostyler/geostyler/blob/master/)
- [x] I have followed the [guidelines for contributing](https://github.com/geostyler/geostyler/blob/master/CONTRIBUTING.md)
- [x] The proposed change fits to the content of the [code of conduct](https://github.com/geostyler/geostyler/blob/master/CODE_OF_CONDUCT.md)
- [ ] I have added or updated tests and documentation, and the test suite passes (run `npm test` locally)
- [ ] I'm lost; why do I have to check so many boxes? Please help!

### CODE REVIEW

**Comment by jansule** [comment]
Thanks @pragmaeuge 

Could you please remove the changes in the `package-lock.json`. Even though it makes sense to update to lockfile version 2, this should rather be part of a separate PR.

Also it looks like your editor autoformatted the files you edited. Could you please undo these changes, so that only your actual changes can be reviewed here?

File: src/Util/OlStyleUtil.ts [hunk 1]
```diff
@@ -147,13 +147,15 @@ class OlStyleUtil {
...
   public static getTextFont(symbolizer: TextSymbolizer) {
-    // since TextSymbolizer has no prop for font weight we use 'Normal' as default
-    const weight = 'Normal';
+    const fontWeight = symbolizer.fontWeight ?? "normal";
+    const fontStyle = symbolizer.fontStyle ?? "normal";
+
     const size = symbolizer.size;
     const font = symbolizer.font;
-    return weight + ' ' + size + 'px ' + font;
+    return fontWeight + " " + fontStyle + " " + size + "px " + font;
```
Comment (jansule) [inline src/Util/OlStyleUtil.ts:155]:
```suggestion
    return fontWeight + ' ' + fontStyle + ' ' + size + 'px ' + font;
```

Just a nitpick

**Comment by pragmaeuge** [comment]
Hi, 
can you check if now is ok?

**Review by jansule — COMMENTED** [review]
This looks good. Thanks @pragmaeuge! Could you make sure that tests succeed?

File: src/Util/OlStyleUtil.ts [hunk 2]
```diff
@@ -147,13 +147,15 @@ class OlStyleUtil {
    * @param symbolizer The TextSymbolizer to derive the font string from
    */
   public static getTextFont(symbolizer: TextSymbolizer) {
-    // since TextSymbolizer has no prop for font weight we use 'Normal' as default
-    const weight = 'Normal';
+    const fontWeight = symbolizer.fontWeight ?? "normal";
```
Comment (jansule) [inline src/Util/OlStyleUtil.ts:150]: Nitpick
```suggestion
    const fontWeight = symbolizer.fontWeight ?? 'normal';
```

File: src/Util/OlStyleUtil.ts [hunk 3]
```diff
@@ -147,13 +147,15 @@ class OlStyleUtil {
...
+    const fontStyle = symbolizer.fontStyle ?? "normal";
```
Comment (jansule) [inline src/Util/OlStyleUtil.ts:151]: Nitpick
```suggestion
    const fontStyle = symbolizer.fontStyle ?? 'normal';
```

**Review by jansule — COMMENTED** [review]
Tests still seem to fail. Do they succeed, when you run them locally with `npm run test`?

**Comment by pragmaeuge** [comment]
Hi,
now it should be ok

**Review by jansule — APPROVED** [review]
Very nice! Thanks @pragmaeuge

**Pull request merged**